Modix.Services/Starboard/StarboardHandler.cs

```diff
@@ -31,80 +34,76 @@ public Task Handle(ReactionRemoved notification, CancellationToken cancellationT
 
         private async Task HandleReaction(Cacheable<IUserMessage, ulong> cachedMessage, SocketReaction reaction)
         {
-            if (!_service.IsStarReaction(reaction))
+            var emote = reaction.Emote;
+            if (!_service.IsStarEmote(emote))
             {
                 return;
             }
 
             var message = await cachedMessage.GetOrDownloadAsync();
-            if (!(message.Channel is IGuildChannel channel))
+            if (!(message.Channel is IGuildChannel channel)
```

Comment:
Readability wise, consider changing this to `message.Channel != channel` or something alike. Currently it reads "not message channel equals channel" rather than "message channel isnt channel".

Reply:
I *wish* `is` had a "not" variation. As is, that expression is checking for type and then assigning to a new local named `channel`. Not sure there's a better way to rewrite without pulling apart the expression